src/analysis/normed_space/linear_isometry.lean

```diff
@@ -130,6 +130,26 @@ instance : monoid (E →ₗᵢ[R] E) :=
 
 end linear_isometry
 
+namespace submodule
+
+variables {R' : Type*} [ring R'] [module R' E] (p : submodule R' E)
+
+/-- `submodule.subtype` as a `linear_isometry`. -/
+def subtype_isometry : p →ₗᵢ[R'] E := ⟨p.subtype, λ x, rfl⟩
+
+@[simp] lemma coe_subtype_isometry : ⇑p.subtype_isometry = p.subtype := rfl
+
+@[simp] lemma subtype_isometry_to_linear_map : p.subtype_isometry.to_linear_map = p.subtype := rfl
+
+/-- `submodule.subtype` as a `continuous_linear_map`. -/
+def subtype_continuous : p →L[R'] E := p.subtype_isometry.to_continuous_linear_map
```

Comment:
I don't like this name, because it seems to be saying that `subtype` is continuous. `subtype_clm`? `subtype_L`?

Reply:
Renamed to `submodule.subtypeL`.